### PR TITLE
README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by klumb**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# CHAOSS Website - Content, Issues, Requests and Edits
+
+The CHAOSS website is a Wordpress site.
+
+## Please use the Issues Tab to submit content requests, edits, and enhancements.
```
Comment (GeorgLink) [inline CONTRIBUTING.md]: Why not pull requests?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## News and blog posts
+To add news and blogs articles, please create a Markdown file in the news folder. When accepted, the content will be added to the website.
+
+## Wordpress/Github Markdown Content
+
+The site is built using GitHub content areas. Editing content in the associated GitHub markdown files will edit content on the Wordpress production site.
+
+Editable content are in folders in this repo and are named for the websites key navigation pages. Within these folders are pages and content areas in markdown files. The content areas are named using the header for the content section.
+
+To edit these content areas, please fork the repo and make changes to the markdown files.
```
Comment (GeorgLink) [inline CONTRIBUTING.md]: ... and create a pull request. 

-- Without the pull request, we will not know about the changes.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+For questions, please email the site [maintainers](README.md).
```
Comment (GeorgLink) [inline CONTRIBUTING.md]: Looks like we need a new section "Maintainers" to use the anchor as the link here.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# CHAOSS Website - Content, Issues, Requests and Edits
+
+The CHAOSS website is a Wordpress site.
```
Comment (GeorgLink) [inline CONTRIBUTING.md]: Add here that content is automatically included from this repository.

File: CONTRIBUTING.md (hunk 2 above)
Comment (GeorgLink) [reply CONTRIBUTING.md]: Maybe link to a .github/CONTRIBUTING.md file like other repos.

Comment (klumb) [reply CONTRIBUTING.md]: This was implied.  I changed it to explicit.

File: CONTRIBUTING.md (hunk 1 above)
Comment (klumb) [reply CONTRIBUTING.md]: Redundant

File: CONTRIBUTING.md (hunk 4 above)
Comment (klumb) [reply CONTRIBUTING.md]: Added

File: CONTRIBUTING.md (hunk 3 above)
Comment (klumb) [reply CONTRIBUTING.md]: I added this

**Comment by klumb** [comment]
I am merging based on @GeorgLink feedback.

**Pull request merged**